group_vars/all

```diff
@@ -66,3 +66,11 @@ calico_installation_type: manifest
 
 ##### List of images to be prepulled #####
 prepull_images: []
+
+##### ETCD version #####
+etcd_version: v3.5.9
+
+##### CNI Configurations #####
+cni_plugins_version: v1.3.0
+cni_plugins_url: https://github.com/containernetworking/plugins/releases/download
+cni_plugins_tarball: "cni-plugins-linux-ppc64le-{{ cni_plugins_version }}.tgz"
```

Comment:
see if you can avoid hardcoding ppc64le and use ansible variable to decide the arch